Downloads/kubeconfig

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
Oops inclusion?

Reply:
I have a habit of storing scripts, examples, etc in the same dir as the project (under `/examples`, or `/dev`).  Global `.gitignore` has worked well for me: https://help.github.com/en/github/using-git/ignoring-files